dockerfiles/csharp/restore.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+find `pwd` -name "*.sln" -exec dotnet restore "{}" \; -exec nuget restore "{}" \;
```

Comment:
Visual Studio solution files don't declare dependencies, they just bundle projects. Visual Studio Code for example doesn't use solution files. Dependencies are declared in .csproj or older project.json (which needs to be converted with `dotnet migrate`)